5.5/Dockerfile

```diff
@@ -27,30 +27,26 @@ EXPOSE 3306
 # to make sure of that.
 RUN rpmkeys --import file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 && \
     yum -y --setopt=tsflags=nodocs install https://www.softwarecollections.org/en/scls/rhscl/mysql55/epel-7-x86_64/download/rhscl-mysql55-epel-7-x86_64.noarch.rpm && \
-    yum -y --setopt=tsflags=nodocs install gettext hostname bind-utils mysql55 && \
+    yum -y --setopt=tsflags=nodocs install gettext hostname bind-utils which mysql55 && \
     yum clean all && \
     mkdir -p /var/lib/mysql/data && chown -R mysql.0 /var/lib/mysql && \
     test "$(id mysql)" = "uid=27(mysql) gid=27(mysql) groups=27(mysql)"
 
-COPY run-*.sh /usr/local/bin/
-COPY contrib /var/lib/mysql/
-
-# Loosen permission bits for group to avoid problems running container with
-# arbitrary UID
-# When only specifying user, group is 0, that's why /var/lib/mysql must have
-# owner mysql.0; that allows to avoid a+rwx for this dir
-RUN chmod -R g+rwx /var/lib/mysql
-
 # When bash is started non-interactively, to run a shell script, for example it
 # looks for this variable and source the content of this file. This will enable
 # the SCL for all scripts without need to do 'scl enable'.
-ENV BASH_ENV=/var/lib/mysql/scl_enable \
-    ENV=/var/lib/mysql/scl_enable \
-    PROMPT_COMMAND=". /var/lib/mysql/scl_enable"
+ENV BASH_ENV=/usr/share/container-scripts/mysql/scl_enable \
+    ENV=/usr/share/container-scripts/mysql/scl_enable \
+    PROMPT_COMMAND=". /usr/share/container-scripts/mysql/scl_enable" \
+    ENABLED_COLLECTIONS=mysql55
+
+ADD root /
```

Comment:
@hhorak I do believe this change will go to all other images as well and this is just first PR to gather feedback.